Daniel-L-Ross/kennels
# Dr routing navbar

# Description

-Implemented a Navbar for the constant navigation elements.
-Created a ApplicationViews component to define what each "page" of the website displays
-changed the main Kennel.js application to display to child componenets
-updated index.js to use react-router-dom

Fixes #3 

## Type of change

Please delete options that are not relevant.



# Testing Instructions

git fetch --all
git checkout dr-routing-navbar
npm start

Each link in the navbar should render new content to the dom. 



# Checklist:
